sklearn/tests/test_docstring_parameters.py

```diff
@@ -269,7 +269,8 @@ def test_fit_docstring_attributes(name, Estimator):
         est.fit(X, y)
 
     skipped_attributes = {'x_scores_',  # For PLS, TODO remove in 1.1
-                          'y_scores_'}  # For PLS, TODO remove in 1.1
+                          'y_scores_',  # For PLS, TODO remove in 1.1
+                          'feature_names_in_'}  # Ignore for now
```

Comment:
are we gonna fix this in this PR? We don't have to, but if we don't should create a separate issue for a sprint or something.